charmcraft/commands/store/store.py

```diff
@@ -76,3 +88,29 @@ def list_registered_names(self):
         for item in response['charms']:
             result.append(Charm(name=item['name'], private=item['private'], status=item['status']))
         return result
+
+    def upload(self, name, filepath):
+        """Upload the content of filepath to the indicated charm."""
+        upload_id = self._client.push(filepath)
+
+        endpoint = '/v1/charm/{}/revisions'.format(name)
+        response = self._client.post(endpoint, {'upload-id': upload_id})
+        status_url = response['status-url']
+        logger.debug("Upload id indicated to the Store, got status url %s", status_url)
```

Comment:
I'm not sure what `Upload id indicated` means. Also "to the store" seems to me to not add anything to the message. How about `"Upload %s started, got status url %s", upload_id, status_url`?

Reply:
of course it's not 'started'. Hrmph.

Reply:
I like it! Improved.

Also found that I forgot to test logs, did that now.